pkgs/tools/system/smartmontools/default.nix

```diff
@@ -23,15 +22,17 @@ in stdenv.mkDerivation rec {
     sha256 = "1mlc25sd5rgj5xmzcllci47inmfdw7cp185fday6hc9rwqkqmnaw";
   };
 
-  patches = [ ./smartmontools.patch ];
+  patches = [
+    # fixes darwin build
```

Comment:
Added a comment here patch following the suggestion from `nixpkgs-review`.

Reply:
The suggestion is from nixpkgs-hammering.

Reply:
Today I learned